### PR TITLE
Some Civil and Tech stuff, fix mod startup

### PR DESCRIPTION
**Pull request by tomsons26**
Add some civil buildings, the E3 demo map where all can be seen, add Civil and Tech building defaults

### CODE REVIEW

File: rules/civilian.yaml [hunk 1]
```diff
@@ -1,6 +1,208 @@
...
+		Range: 6c0
+	Turreted:
+		ROT: 255
+	BodyOrientation:
+		QuantizedFacings: 8
+	Cargo:
+		Types: Infantry
+		MaxWeight: 5
+		PipCount: 5
+#		InitialUnits: e1, e2
```
Comment (Phrohdoh) [inline rules/civilian.yaml]: These lines need to be removed.

**Comment by abcdefg30** [comment]
It would be nice, if you could correct the bounds.

The death animations aren't 100% correct as well, but this can be done in a future PR.

File: rules/civilian.yaml [hunk 2]
```diff
@@ -1,6 +1,207 @@
+# White House
```
Comment (abcdefg30) [inline rules/civilian.yaml]: Please remove this comment.

File: rules/civilian.yaml [hunk 3]
```diff
@@ -1,6 +1,207 @@
...
+	Tooltip:
+		Name: White House
+	Health:
+		HP: 1000
+	Armor:
+		Type: Concrete
+	RevealsShroud:
+		Range: 6c0
+		
+# Wash Building 3
```
Comment (abcdefg30) [inline rules/civilian.yaml]: And this one.

File: rules/civilian.yaml [hunk 4]
```diff
@@ -1,6 +1,207 @@
...
+cawash03:
+	Inherits: ^CivBuilding
+	Building:
+		Footprint: xxx xxx xxx xxx  # OR xxxx xxxx xxxx
```
Comment (abcdefg30) [inline rules/civilian.yaml]: This needs to be resolved as well.

File: rules/civilian.yaml [hunk 5]
```diff
@@ -1,6 +1,207 @@
...
+		HP: 1000
+	Armor:
+		Type: Steel
+	RevealsShroud:
+		Range: 6c0
+
+cawash05:
+	Inherits: ^CivBuilding
+	Building:
+		Footprint: xx xx xx xx  # OR xxxx xxxx
```
Comment (abcdefg30) [inline rules/civilian.yaml]: Same here.

**Comment by abcdefg30** [comment]
Thanks.

**Pull request merged**